### PR TITLE
Remove Legacy application Routing Link from Introduction Pages

### PR DESCRIPTION
**Pull request by bradbergeron-us**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
This Pull Request makes use of the showMebEnhancements flipper flag to conditionally render the link on the introduction page for 1990EZ form. The user will be directed to the paper form if the flag is turned on. If the flag is turned off, the user will be re-directed to the paper application.

## Testing done
Manual Testing - Navigating to the Homepage of the application http://localhost:3001/education/apply-for-benefits-form-22-1990/introduction and verifying the link on the bottom of the page based on the feature flag status.

## Screenshots
_Note: This field is mandatory for component work and UI changes (non-component work should NOT have screenshots)._

PaperForm
image-2023-03-17-10-17-45-123
hyperLinkUpdateParagraph

featureToggleFALSE
enhancementsFlipperFeature
## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [X]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X]  No error nor warning in the console.
- [X]  Events are being sent to the appropriate logging solution
- [X]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [X]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [X]  I added a screenshot of the developed feature
- [X]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by nathanielbuck — APPROVED** [review]

**Review by btruongafs — APPROVED** [review]

File: src/applications/my-education-benefits/components/IntroductionLoginV1.jsx [hunk 1]
```diff
@@ -30,27 +33,35 @@ function IntroductionLoginV1({ firstName, eligibility, route, user }) {
...
       )}
-
       {!user?.login?.currentlyLoggedIn && (
-        <a href="/education/apply-for-education-benefits/application/1990/applicant/information">
-          Start your application without signing in
-        </a>
+        <>
+          {showMebEnhancements ? (
+            <a href="https://www.va.gov/find-forms/about-form-22-1990/">
+              If you don’t want to sign in, you can apply using the paper form.
```
Comment (btruongafs) [inline src/applications/my-education-benefits/components/IntroductionLoginV1.jsx:40]: @bradbergeron-us this still has the hyperlink on the entire sentence

**Review by btruongafs — CHANGES_REQUESTED** [review]
doesnt look like the hyperlink got changed to just the 'paper form' copy only

File: src/applications/my-education-benefits/components/IntroductionLoginV1.jsx
Comment (bradbergeron-us) [inline src/applications/my-education-benefits/components/IntroductionLoginV1.jsx:1]: I updated this file, but it is not currently being used. The Introduction page container only imports InroductionLoginV2.jsx so the V1 is not used. @btruongafs

File: src/applications/my-education-benefits/components/IntroductionLoginV2.jsx [hunk 2]
```diff
@@ -88,15 +81,26 @@ function IntroductionLoginV2({
                 </button>
               </div>
             </va-alert>
-
             <p className="vads-u-margin-top--4">
-              <a href="/education/apply-for-education-benefits/application/1990/applicant/information">
-                Start your application without signing in
-              </a>
+              {showMebEnhancements ? (
```
Comment (bradbergeron-us) [inline src/applications/my-education-benefits/components/IntroductionLoginV2.jsx:85]: @btruongafs, the essential part of the change occurs here in the IntroductionLoginV2.jsc component, which the Introduction Page Container renders. Lines 85-100 address changing the copy and conditionally rendering based on the showMebEnhancements feature flag being toggled on or off.

**Review by btruongafs — APPROVED** [review]

**Pull request merged**